packages/playground-ui/theme.css

```
@@ -389,8 +389,7 @@ html.light {
   --shadow-card: 0 2px 8px oklch(0 0 0 / 0.4);
   --shadow-elevated: 0 8px 24px oklch(0 0 0 / 0.5);
 
-  /* Drop-shadow only — no white inset: @theme inlines values, so it can't theme per-mode and reads dirty on dark. */
-  --shadow-dialog: 0 8px 24px -8px oklch(0% 0 0deg / 28%), 0 24px 56px -16px oklch(0% 0 0deg / 36%);
+  --shadow-dialog: 0 1px 2px -1px oklch(0% 0 0deg / 12%), 0 16px 40px -20px oklch(0% 0 0deg / 22%);
```

Stylelint (17.14.0)

[error] Expected no empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)